### ISSUE TITLE
Update the version of the react-scripts in react template

### ISSUE THREAD

**Issue by endo-aki22** [codesandbox/codesandbox-client#6833]
If anyone installs the react-template and tailwindcss, the css is not loaded.
Because the default template uses old version of the react-scripts.
![2022-08-20_04-06](https://user-images.githubusercontent.com/61251512/185699315-55877036-f236-4745-917b-c7567ac1fb3a.png)
It needs to keep the latest version of react-scripts.

**Comment by JamesACS** [comment]
Hey @endo-aki22 -

You can upgrade the script version by clicking on it.

**Comment by endo-aki22** [comment]
@JamesACS 
yes. you are right. but I guess most people don't concern about it. because as I had ever faced, if anybody goes through an invite link and the project has many dependencies, he wouldn't check the version. maybe only see the version of `react`.
